template.yaml

```diff
@@ -2931,4 +2931,56 @@ Resources:
               detail:
                 responsePayload:
                   topic:
-                    - PublicationService.Resource.Deleted
\ No newline at end of file
+                    - PublicationService.Resource.Deleted
+
+  DynamodbResourceBatchJobDLQ:
+    Type: AWS::SQS::Queue
+    Properties:
+      MessageRetentionPeriod: 1209600
+      VisibilityTimeout: 60
+
+  DynamodbResourceBatchJobWorkQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      VisibilityTimeout: 360 # 6 times the Lambda timeout
+      MessageRetentionPeriod: 1209600 # 14 days
+      ReceiveMessageWaitTimeSeconds: 20
+      RedrivePolicy:
+        deadLetterTargetArn: !GetAtt DynamodbResourceBatchJobDLQ.Arn
+        maxReceiveCount: 3
+      RedriveAllowPolicy:
+        redrivePermission: byQueue
+        sourceQueueArns:
+          - !GetAtt DynamodbResourceBatchJobDLQ.Arn
+
+  DynamodbResourceBatchJobFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: no.unit.nva.publication.events.handlers.batch.dynamodb.DynamodbResourceBatchJobHandler::handleRequest
+      CodeUri: publication-event-handlers
+      Policies:
+        - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+        - !GetAtt ResourceServiceAccessManagedPolicy.PolicyArn
+        - !GetAtt EventsLambdaManagedPolicy.PolicyArn
+      Timeout: 60 # if changing, also change the visibility timeout of the SQS queue
```

Comment:
nitpick: These timeouts and retention periods etc should probably be pinned in a top-level parameter with a default value, because I assume we usually want them to be the same for all queues/lambdas.

Reply:
hmmm I'm not sure. maybe all in some groups, like endpoints, handlers, etc, but still I'm not sure. Default lambda should probably be 20 like it is now and this is an exception?